packages/appstore/src/components/onboarding/onboarding.tsx

```diff
@@ -49,17 +51,56 @@ const Onboarding = ({ contents }: TOnboardingProps) => {
                     <Text as='p' size='xs' align='center' className='onboarding-footer-text'>
                         {contents[onboarding_step as keyof typeof contents]?.footer_text}
                     </Text>
-                    <div className='onboarding-footer-buttons'>
-                        <Button secondary onClick={prevStep} style={step === 1 ? { visibility: 'hidden' } : {}}>
-                            {localize('Back')}
-                        </Button>
-                        <ProgressBarOnboarding step={step} amount_of_steps={number_of_steps} setStep={setStep} />
-                        <Button primary onClick={nextStep}>
-                            {contents[onboarding_step as keyof typeof contents]?.has_next_content
-                                ? contents[onboarding_step as keyof typeof contents]?.next_content
-                                : localize('Next')}
-                        </Button>
-                    </div>
+                    {isDesktop() && (
```

Comment:
isn't better to use `<DesktopWrapper> and <MobileWrapper>` instead?

Reply:
I have tried it but did not get the results that I wanted. So I used isDesktop and isMobile in order to move the card forward faster.